cypress/e2e/rich-text/RichTextEditor.Tables.spec.ts

```diff
@@ -177,10 +173,15 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
       );
     });
 
-    it('does not delete table header cells when selecting the whole table', () => {
+    it('does not delete table header cells when selecting the whole header row', () => {
```

Comment:
I'd argue this is now a completely different test and probably worth a follow up to see if it's even needed.

Reply:
Totally agree, will sync with @colomolo on this tomorrow